#pragma GCC is not allowed inside functions with GCC < 4.3
and -Wmaybe-uninitialized is not recognized by clang
https://sourceforge.net/p/isync/bugs/36/
--- src/util.c.orig	2017-08-05 14:57:19.000000000 -0500
+++ src/util.c	2017-10-04 00:22:12.000000000 -0500
@@ -519,15 +519,7 @@
 		for (ll = 0; ll < inl; ll++)
 			if (arg[i + ll] != in[ll])
 				goto rnexti;
-#ifdef __GNUC__
-# pragma GCC diagnostic push
-/* https://gcc.gnu.org/bugzilla/show_bug.cgi?id=42145 */
-# pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
-#endif
 		memcpy( p, out, outl );
-#ifdef __GNUC__
-# pragma GCC diagnostic pop
-#endif
 		p += outl;
 		i += inl;
 		continue;
